Production_scripts/Process_extracts/Produce_extract_tests/extract_sds_tests.R

```diff
@@ -0,0 +1,40 @@
+####################################################
+# Name of file - extract_sds_tests.R
+# Original Authors - Zihao Li
+# Original Date - November 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               sds files
+#####################################################
+
+# Load packages
+library(createslf)
+
+# Read in Data-----------------------------------------
+
+year <- check_year_format("1920")
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "SDS")) %>%
+  data.table::as.data.table() %>%
+  mutate("year" = year) %>%
+  as_tibble()
```

Comment:
```suggestion
# Read new data file
new_data <- readr::read_rds(get_source_extract_path(year, "SDS")) %>%
```
We can remove these lines as this was an oversight on my part. The year specific SDS extract should have had the variable `year` included in the processed extract. I've now corrected this and ran the file again to produce the corrected rds file which this should now be working with the code